using-mobile-extensions/adobe-target/README.md

```diff
@@ -42,20 +42,28 @@ You can find your Target client code by going to the Setup tab and selecting Imp
 {% endtab %}
 
 {% tab title="iOS" %}
+Add the Target library to your project via your `Podfile` by adding `pod 'ACPTarget'`
 #### Objective-C
 
-1. Add the Target library to your project via your `Podfile` by adding `pod 'ACPTarget'`
-2. Import the Target and Identity library. 
+Import the Target and Identity library. 
 
    ```objectivec
-   #import <ACPCore_iOS/ACPCore_iOS.h>
-   #import <ACPTarget_iOS/ACPTarget_iOS.h>
-   #import <ACPIdentity_iOS/ACPIdentity_iOS.h>
+   #import "ACPCore.h"
+   #import "ACPTarget.h"
+   #import "ACPIdentity.h"
+   #import "ACPTargetRequestObject.h"
+   #import "ACPTargetPrefetchObject.h"
    ```
 
 #### Swift
 
-Update the bridging header
+   ```swift
+   #import ACPCore
+   #import ACPTarget
+   #import ACPIdentity
+   #import "ACPTargetRequestObject.h"
```

Comment:
@phuo1032 These imports have red background, is this intended?

Reply:
I think the issue is that the bridging header file is an objective-c file and the code snippet is marked as swift. Marking the code snippet as objective-c should fix this.

Reply:
Also, the other imports in the bridging header would be "ACPCore.h", "ACPTarget.h", and "ACPIdentity.h"

Reply:
the way this works in swift depends on whether a bridging-header is needed.  

in short, if `ACPCore` is a framework, then the appropriate way to integrate is in source, with `#import ACPCore`.  
if `ACPCore` is a static library, then you must include `#import "ACPCore.h"` in a bridging-header, and in the swift source you will not need to include any `#import` statements.

hold off on these changes for now, as we're evaluating some options on how we distribute our pods for swift